### DISCUSSION TITLE
What is your lab in NAIST? あなたの所属研究室は？

### DISCUSSION THREAD

**Discussion by TaiseiURAKAMI** [NAIST-SE/AttractingContributors#470] [General]
Please let me know about your laboratory in NAIST. 
あなたのNAISTの所属研究室を教えてください。
I am in the Network Systems Laboratory. 
私は、ネットワークシステム学研究室です。

Please follow this link to join us.
こちらのリンクから、参加してください。
https://github.com/TaiseiURAKAMI/AttractingContributors

**Comment by aru1702** [comment]
こんにちは！
インタラクティブメディア設計研究室からです
よろしくねー

See my repo: https://github.com/aru1702/AttractingContributors-TouhouFumo
Hop into my discussion: https://github.com/NAIST-SE/AttractingContributors/discussions/407

**Comment by XC1996114514** [comment]
初めまして
光メディアインタフェース研究室のチン　ケン　です
よろしくお願いいたします。

So could you help me to filled the readme?
Here is my repository;
https://github.com/XianChen1996/AttractingContributors_type_in_your_favourite_footballteam

Thanks a lot for helping me!

**Comment by sh1n00** [comment]
Hello My name is Riku Shinohara!

Update your repo and make pull req. pls check it later.

help me to making a pull request!

https://github.com/namekosiru/FavoriteSoftwareService

Thank your effort!
